datahub-web-react/src/utils/customGraphQL/useGetEachEntitySearchResults.ts

```diff
@@ -0,0 +1,44 @@
+import { EntityType } from '../../types.generated';
+import { useGetSearchResultsQuery } from '../../graphql/search.generated';
+
+export function useGetEachEntitySearchResults(input: any) {
```

Comment:
nice! 

very minor nit: 

can we call this "useGetAllEntitySearchResults"

Reply:
Done!